### ISSUE TITLE
table overrides not created

### ISSUE THREAD

**Issue by Flachdachs** [ThePsionic/RSWikiLinker#32]
When starting the bot with a blank database and with no invites then the table `guilds` is created when the first invite happens in the event `guildCreate` in the `catch` callback. But the table `overrides` is not created. And it will not be created anywhere else. In the next `ready` event the table `guilds` can be accessed, so the `catch` callback doesn't create `overrides`. When `overrides` doesn't exists the bot doesn't answer to requests, but creates an error message in the log.

tl;dr the `catch` callback for event `guildCreate` needs a table create statement for `overrides` like in the `ready` event.
